src/components/AppRouting.js

```diff
@@ -0,0 +1,32 @@
+import {
+  Home,
+  About,
+  Services,
+  Team,
+  Appointment,
+  FAQ,
+  Contact,
+  PersonalDataProtection,
+  PrivacyPolicy,
+} from "../pages/index";
```

Comment:
when importing from "./xxxx/index" you don't need "index", it's default so "./xxx" is enough :)

Reply:
Changed import.